### DISCUSSION TITLE
Question about the screen reader logical reading order in dynamic page layout header section

### DISCUSSION THREAD

**Discussion by anuvenkatesh1** [UI5/webcomponents-react#3267] [Q&A] [answered]
One of our pages use the [dynamic page layout component](https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-dynamicpage--default-story), which has the object header and content part just above the table. 
The reading order is Title & Status (1, 2) -> all buttons -> then the other details (location/availability etc) are read in screen reader Browse mode. May be in the context of this test page is ok. 
 

However, in the context of our page ( its about goals that we create and this page has the goal details. The object page header summarizes the details) and so it would make sense for a screen reader user to read all the details on the left first before moving on to the actions that can be taken on them ie in the context of the above dynamic page layout component, we want the title/status/updated/location/availability etc to be read first, before reading the buttons.
Please note that in our page we do not have the "collapse" button to collapse the header content area.

I'm unable to attach screenshots. Hope the context is clear. Should we use a different component? If so what would be the component? or in the current component is there any workaround to achieve this?

**Comment by Lukas742** [comment, marked as answer]
Discussed in #3287

Locked.
